ui/app/routes/vault/cluster/metrics/http-requests.js

```diff
@@ -2,6 +2,6 @@ import ClusterRouteBase from '../cluster-route-base';
 
 export default ClusterRouteBase.extend({
   model() {
-    return this.store.queryRecord('requests', {});
+    return this.store.queryRecord('http-requests', {});
```

Comment:
wanna change this to `metrics/http-requests` here so the http requests page doesn't break?

Reply:
Good idea.